packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -274,7 +275,7 @@ contract CrabMigration is Ownable {
 
             uint256 crabV2Amount = CrabStrategyV2(crabV2).balanceOf(address(this));
             // send back V2 tokens to the user
-            CrabStrategyV2(crabV2).transfer(_initiator, crabV2Amount);
+            CrabStrategyV2(crabV2).transfer(_initiator, crabV2Amount.sub(initialCrabAmount));
             IERC20(wPowerPerp).transfer(_initiator, IERC20(wPowerPerp).balanceOf(address(this)));
 
             uint256 excessEth = address(this).balance;
```

Comment:
we need to update the events to use crabV2Amount.sub(initialCrabAmount), either like that or as a variable that is saved (prb better to do second one)

emit FlashMigrate

Reply:
ie ```uint256 crabV2ToTransfer = CrabStrategyV2(crabV2).balanceOf(address(this)).sub(initialCrabAmount);``` and use that variable in the event and the transfer